Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -682,6 +713,47 @@ namespace O3DE::ProjectManager
         
     }
 
+    void ProjectsScreen::HandleOpenProjectExportSettings(const QString& projectPath)
+    {
+        AZ::Outcome<EngineInfo> engineInfoResult = PythonBindingsInterface::Get()->GetProjectEngine(projectPath);
+        AZ::Outcome projectBuildPathResult = ProjectUtils::GetProjectBuildPath(projectPath);
+
+        auto engineInfo = engineInfoResult.TakeValue();
+        auto buildPath = projectBuildPathResult.TakeValue();
+
+        QString projectName = tr("Project");
+        auto getProjectResult = PythonBindingsInterface::Get()->GetProject(projectPath);
+        if (getProjectResult)
```

Comment:
Shouldn't we handle the error case if there is a failure when getting the project information? If it fails here, it will fail in the eventual `o3de.bat export-project` command as well.